assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -77,49 +97,47 @@ function Media3pPane(props) {
 
   const {
     searchTerm,
+    selectedProvider,
     setSelectedProvider,
     setSearchTerm,
-    unsplash,
-    selectedProviderState,
+    media3p,
   } = useMedia(
     ({
       media3p: {
         state: { selectedProvider, searchTerm },
         actions: { setSelectedProvider, setSearchTerm },
-        unsplash,
       },
       media3p,
     }) => ({
       searchTerm,
+      selectedProvider,
       setSelectedProvider,
       setSearchTerm,
-      unsplash,
-      selectedProviderState: media3p[selectedProvider ?? ProviderType.UNSPLASH],
+      media3p,
     })
   );
 
-  const {
-    state: { categories },
-    actions: { selectCategory, deselectCategory },
-  } = selectedProviderState;
-
   useEffect(() => {
-    if (isActive) {
-      setSelectedProvider({ provider: 'unsplash' });
+    if (isActive && !selectedProvider) {
+      setSelectedProvider({ provider: ProviderType.UNSPLASH });
```

Comment:
Should it be whatever is first in the providers config?

Reply:
Good catch.